### PR TITLE
Added verbiage to identify fields as required

### PR DESCRIPTION
**Pull request by elcazador**
see [#1965](https://github.com/CityofOttawa/ottawa_profile/issues/1965).

Adds a required indicator on the fields.

### CODE REVIEW

File: css/oembed.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.oembed-required {
+    color: #B40000 !important;
```
Comment (budimanc) [inline css/oembed.css:2]: Optional: Maybe apply the same hex code (#f00) that Drupal uses (for consistency)?

**Review by budimanc — APPROVED** [review]
Looks good and works great (I added a minor optional suggestion though).

**Review by DaveBagler — APPROVED** [review]

File: css/oembed.css (hunk 1 above)
Comment (elcazador) [reply css/oembed.css:2]: That colour code is the one we have defined in the sass variables file.

Comment (budimanc) [reply css/oembed.css:2]: It seems we use two shades of red then.  I was looking at the "backend" when editing a node, which uses the Drupal's theme system .form-required #f00.   The #b40000 is used in webform which we customized in our sass file.

image

**Pull request merged**